### PR TITLE
Reduce whitespace in experiment view & others

### PR DESCRIPTION
**Pull request by smurching**
This PR improves space utilization on the experiment view - the changes made to do so end up improving space utilization on other pages too. The metric view does look a little strange (i.e. the plots look more left-aligned now), but I'd say most of the other pages have strictly improved

#### Experiment page
Before:
image

After:
image

#### Run detail page
Before:
image

After:
image

#### Metrics page
Before:
image

After:
image

#### Run comparison page
Before:
image

After:
image

### CODE REVIEW

File: mlflow/server/js/src/components/ArtifactView.css [hunk 1]
```diff
@@ -32,6 +32,7 @@ div.artifact-view {
 }
 
 .empty-artifact-outer-container {
+  width: calc(100% - 128px);
```
Comment (aarondav) [inline mlflow/server/js/src/components/ArtifactView.css]: magical

File: mlflow/server/js/src/components/ExperimentRunsTableCompactView.js [hunk 2]
```diff
@@ -414,7 +414,7 @@ class ExperimentRunsTableCompactView extends PureComponent {
                 width={
                   Math.max(width, tableMinWidth)
                 }
-                height={Math.max(height - TABLE_HEADER_HEIGHT, 200)}
+                height={Math.max(height - TABLE_HEADER_HEIGHT, 200) + 20}
```
Comment (aarondav) [inline mlflow/server/js/src/components/ExperimentRunsTableCompactView.js]: oo kay

File: mlflow/server/js/src/components/HomeView.js [hunk 3]
```diff
@@ -59,7 +59,6 @@ class HomeView extends Component {
              <NoExperimentView/>
             }
           </div>
-          <div className="experiment-view-right"/>
```
Comment (aarondav) [inline mlflow/server/js/src/components/HomeView.js]: Not needed?

File: mlflow/server/js/src/components/RunView.css [hunk 4]
```diff
@@ -72,7 +71,7 @@ i.expander {
 }
 
 textarea.run-command {
-  width: 100%;
+  width: calc(100% - 108px);
```
Comment (aarondav) [inline mlflow/server/js/src/components/RunView.css]: obviously 108px

**Comment by aarondav** [comment]
This change enables makes my laptop to render 15.5 min-width rows as opposed to 12.1. ~25% increase in total table space for my tiny screen - awesome!

File: mlflow/server/js/src/components/ExperimentRunsTableCompactView.js (hunk 2 above)
Comment (smurching) [reply mlflow/server/js/src/components/ExperimentRunsTableCompactView.js]: lol 👍

File: mlflow/server/js/src/components/HomeView.js (hunk 3 above)
Comment (smurching) [reply mlflow/server/js/src/components/HomeView.js]: yep, this element just existed to take up 10% of the screen width on the right side :P - we now take up the whole width minus 64px on each side (hence the magic 128px showing up everywhere)

**Comment by mateiz** [comment]
Are metrics supposed to stay 1 per line in this design? I get stuff like this that is not using the space in the metrics column well:

screen shot 2019-02-21 at 4 15 42 pm

(The same issue also seems to happen with master though).

**Comment by smurching** [comment]
@mateiz good question, unfortunately this PR doesn't improve the layout of metrics/params in the experiment table - that's also possible but may take some more work, so I'd prefer to merge this PR first

**Comment by smurching** [comment]
Ended up pushing one more commit to fix weird behavior where the views would horizontally scroll because their parent container (div with classname "App-content") had a misconfigured width - will merge after tests pass

**Pull request merged**